src/main/scala/xiangshan/backend/MemBlock.scala

```diff
@@ -1930,13 +1929,8 @@ class MemBlockInlinedImp(outer: MemBlockInlined) extends LazyModuleImp(outer)
   pfevent.io.distribute_csr := csrCtrl.distribute_csr
   val csrevents = pfevent.io.hpmevent.slice(16,24)
 
-  val memBlockPerfEvents = Seq(
```

Comment:
Is this signal `memBlockPerfEvents` useless now?

Reply:
Others LGTM

Reply:
Yes, it has been replaced with `perfBlock` below.